_posts/2017-05-17-testing-ghc-release-candidates-with-stack.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Testing GHC release candidates with Stack
+---
+
+Yesterday Ben Gamari [announced the second GHC 8.2.1 release candidate][] on
+behalf of the GHC team. Release candidates allow the community to test new
+versions before they're officially released. Using the new compiler or
+interpreter directly is easy enough, but what about building an entire project?
+How can a Haskell developer test their code with the latest release candidate?
+I'm going to show you how to use Stack to do just that.
+
+Assuming your project already uses Stack, your `stack.yaml` probably looks
+something like this:
+
+``` yaml
+resolver: lts-8.14
+```
+
+You may be using a different resolver or have some other stuff in there. That's
+fine! We're going to update your `stack.yaml` to include information about the
+release candidate. For starters we'll tell it to use the GHC 8.2.1-rc2
+compiler.
+
+``` yaml
+compiler: ghc-8.2.0.20170507
+compiler-check: match-exact
+```
+
+Don't be fooled by the compiler version! `ghc-8.2.0.20160507` is actually GHC
```

Comment:
I think you meant `ghc-8.2.0.20170507` instead of `ghc-8.2.0.20160507` here (note the `2016` instead of `2017`).